recipe/meta.yaml

```diff
@@ -106,9 +105,9 @@ outputs:
         - aiohttp >=3.7
         - aiohttp-cors
         - colorful
-        - gpustat
+        - gpustat >=1.0.0
         - opencensus
-        - prometheus_client >=0.7.1, <0.14.0
+        - prometheus_client >=0.7.1
```

Comment:
why? that constraint is still present in `setup.py`

Reply:
Removing this was part of trying to debug the failures. It can & should be reinstated.

Reply:
OK